src/amo/sagas/addonsByAuthors.js

```diff
@@ -23,7 +23,7 @@ export function* fetchAddonsByAuthors({ payload }) {
       api: state.api,
       filters: {
         addonType,
-        author: authors.join(','),
+        author: authors.sort().join(','),
```

Comment:
It makes the requests a bit more consistent to look at but also helps debugging; if you don't sort the authors it was more of a pain to test things because you have to replicate order everywhere which I didn't at one point and it was a PITA to track down. 😄